SplunkRumWorkspace/SplunkRum/SplunkRum/SplunkRum.swift

```diff
@@ -99,6 +103,16 @@ let SplunkRumVersionString = "0.5.2"
      */
     @objc public var screenNameSpans: Bool = true
 
+    /**
+     The SlowFrame Threshold is an optional configuration
+     */
+    @objc public var slowFrameThreshold: CFTimeInterval = 0.0169
+
+    /**
+     The frozenFrame Threshold is an optional configuration
+     */
+    @objc public var frozenFrameThreshold: CFTimeInterval = 0.7
```

Comment:
Please have the config options as milliseconds

Reply:
Done with threshold config options now in milliseconds.